rename to CTIS (#397)
cmu-delphi/www-main: content/covidcast/surveys.md

---

> title: Delphi's COVID Trends and Impact Surveys (CTIS)

Reviewer: ```suggestion
title: Delphi's COVID-19 Trends and Impact Surveys (CTIS)
```

---

{{< alert category="primary">}}
  Summary results for the survey are now available. [Click here to explore]({{< relref "covidcast/survey-results" >}})
{{< /alert >}}

> In collaboration with Facebook, along with a consortium of universities and public health officials, the [Delphi group]({{< relref "/" >}}) at [Carnegie Mellon University](https://www.cmu.edu/) conducts research surveys to monitor the spread and impact of the COVID-19 pandemic in the United States. This survey is advertised through Facebook. It has run continuously since early April 2020, and about 50,000 people in the United States participate **every day**. Survey results are publicly available on our [CTIS Survey Dashboard]({{< relref "covidcast/survey-results" >}}), [COVIDcast Map]({{< relref "covidcast" >}}), [COVIDcast Exporter]({{< relref "covidcast/export" >}}), and in our [COVIDcast API]({{< apiref "api/covidcast.html" >}}).

Reviewer: ```suggestion
In collaboration with Facebook, along with a consortium of universities and public health officials, the [Delphi group]({{< relref "/" >}}) at [Carnegie Mellon University](https://www.cmu.edu/) conducts the COVID Trends and Impact Survey to monitor the spread and impact of the COVID-19 pandemic in the United States. This survey is advertised through Facebook. It has run continuously since early April 2020, and about 50,000 people in the United States participate **every day**. Survey results are publicly available on our [CTIS Survey Dashboard]({{< relref "covidcast/survey-results" >}}), [COVIDcast Map]({{< relref "covidcast" >}}), [COVIDcast Exporter]({{< relref "covidcast/export" >}}), and in our [COVIDcast API]({{< apiref "api/covidcast.html" >}}).
```

Reviewer: (we didn't mention the name anywhere but the header)

### Media inquiries:

> [Stacy Kish](mailto:skish@andrew.cmu.edu)
> Associate Director, Research Communications
> Dietrich College

Reviewer 2: ```suggestion
[Stacy Kish](mailto:skish@andrew.cmu.edu)  
Associate Director, Research Communications  
Dietrich College  
```

Carnegie Mellon University

### All other questions:

[Alex Reinhart](mailto:areinhar@stat.cmu.edu)
Assistant Teaching Professor

> Department of Statistics & Data Science

Reviewer: The trailing whitespace in the Markdown is actually essential here: two trailing spaces makes Markdown include line breaks instead of merging into one paragraph

> [Alex Reinhart](mailto:areinhar@stat.cmu.edu)
> Assistant Teaching Professor
> Department of Statistics & Data Science

Reviewer 2: ```suggestion
[Alex Reinhart](mailto:areinhar@stat.cmu.edu)  
Assistant Teaching Professor  
Department of Statistics & Data Science  
```
